packages/http-service/src/request.js

```diff
@@ -0,0 +1,25 @@
+import Rx from "rxjs";
+
+import ConnectionManager from "connection-manager";
+import { XHRConnection, ConnectionEvent } from "connections";
+
+export default function request(url, options = {}) {
```

Comment:
We should to either add a third argument or read the connection priority from the options provided to enable high priority requests and streams.

Reply:
Fair point. Things to consider: maybe to control priority we should use raw ConnectionManager instead? Otherwise it feels like it is becoming a leaky abstraction. I'd say `http-service` gives you the default priority and if you need to override it, then you can just import `ConnectionManager` and roll everything manually.

Reply:
💯  Convinced